modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -135,8 +135,9 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
 
             /* Visit_labels
              */
-            $visit_array =Utility::getExistingVisitLabels();
-            $visit_array = array_combine($visit_array, $visit_array);
+            $visit_array   =Utility::getExistingVisitLabels();
```

Comment:
oops; my fault. I would have quickly ran phpcs if you saw it @driusan and pointed it out to me, but am not sure why Travis did not catch this? I really rely on Travis myself.